wasmbinding/bindings/msg.go

```diff
@@ -179,6 +183,11 @@ type BurnTokens struct {
 	BurnFromAddress string `json:"burn_from_address"`
 }
 
+type SetBeforeSendHook struct {
+	Denom        string `json:"denom"`
+	CosmWasmAddr string `json:"cosm_wasm_addr"`
```

Comment:
it's `cosm_wasm_addr` still, not `contract_addr` as we discussed. need to be updated in the integration tests as well